docs/usage/2-manifest.md

```diff
@@ -7,6 +7,13 @@ name: Manifest format
 Projects are defined in `Project.swift` files, which we refer to as manifest files. Manifests need to import the framework `ProjectDescription` which contains all the classes and enums that are available for you to describe your projects. The snippet below shows an example project manifest:
 
 ```swift
+let releaseScheme = Scheme(name: "App-Release",
```

Comment:
Looks like this was added by mistake, the project manifest has an inline scheme

Reply:
I intended to extract Scheme manifest into separate variable for better readability but later have changed it to inline to keep document consistent.